### ISSUE TITLE
No DWDS url on main page

### ISSUE THREAD

**Issue by smnorris** [smnorris/bcdata#31]
For https://catalogue.data.gov.bc.ca/dataset/hydrometric-stations-active-and-discontinued
```
$ bcdata hydrometric-stations-active-and-discontinued
Traceback (most recent call last):
  File "/usr/local/bin/bcdata", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/bcdata/cli.py", line 50, in cli
    driver=driver)
  File "/usr/local/lib/python2.7/site-packages/bcdata/main.py", line 74, in download
    raise ValueError('DWDS URL does not exist, something went wrong')
ValueError: DWDS URL does not exist, something went wrong
```
- there is a link to an additional resource page that includes the DWDS url, is this link consistent in other pages?
- perhaps this is better resolved with using the API as per #25 ?

**Comment by smnorris** [comment]
same result for https://catalogue.data.gov.bc.ca/dataset/bcgs-1-20-000-grid

DWDS url is at https://catalogue.data.gov.bc.ca/dataset/bcgs-1-20-000-grid/resource/4f60db83-6e44-4023-9dfa-9833185a6a41

**Comment by smnorris** [comment]
fixed with 2046eb87ef169c2e48c9c80bf074ce360cc6c952
